besu/src/test/java/org/hyperledger/besu/cli/CommandTestAbstract.java

```diff
@@ -139,15 +139,19 @@ public abstract class CommandTestAbstract {
   private static final Logger TEST_LOGGER = LoggerFactory.getLogger(CommandTestAbstract.class);
 
   protected static final int POA_BLOCK_PERIOD_SECONDS = 5;
+  protected static final int POA_EMPTY_BLOCK_PERIOD_SECONDS =
+      50; // TODO: IS THIS APPLICABLE FOR POA?
```

Comment:
By POA, do you mean `clique`? https://github.com/hyperledger/besu/pull/6082 already delivered the equivalent of your PR for clique, but I think here POA means any proof-of-authority consensus algorithm which includes QBFT, so I think it does make sense to add to this test

Reply:
I added it thinking POA means indeed proof-of-authority, yes... I'll remove the comment.